scripts/upload-compose-secrets.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -eu
+
+if [ "${AWS_ACCOUNT}" = "dev" ]
+   then
+        export PASSWORD_STORE_DIR="${HOME}"/.paas-pass
+        COMPOSE_ACCESS_TOKEN=$(pass "compose/integration_tests/access_token")
+   else
+        export COMPOSE_PASSWORD_STORE_DIR="${HOME}"/.paas-pass-high
```

Comment:
There's some previous context where we've moved them about previously in:

- https://github.com/alphagov/paas-release-ci/pull/30#discussion_r128785439
- https://github.com/alphagov/paas-release-ci/pull/30#discussion_r128794612